### PR TITLE
feat: update currency formatting

### LINKED ISSUE #161: Improvements on PPH-Exchange command
### **As a** 
User

### **I want to** 
Have a properly formatted text response for the pph-exchange command

### **In order to** 
Be able to read properly the conversion response

***

### Acceptance Criteria
- [ ] Text response of the `pph-exchange` command must be properly formatted to follow US or PH locale currency formatting
  - Example: 1000.00 PHP should be displayed as 1,000.00 PHP
- [ ] Update message response from "The exchange rate for M XYZ is N ABC" to "The exchange rate for M XYZ is around N ABC" to provide a more forgiving, accurate response since we're accounting for round offs

### Notes
- Must not affect existing feature
- Current formatting response: 

Image

### PR DESCRIPTION
**Pull request by Jiseeeh**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #161 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Manual testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: bot/cogs/currency_converter.py [hunk 1]
```diff
@@ -82,9 +82,11 @@ async def exchange(
 
         converted_amount = data["result"]
 
-        formatted_from = f"{amount} {from_currency.upper()}"
-        formatted_to = f"{converted_amount} {to_currency.upper()}"
+        amount = float(amount) if "." in amount else int(amount)
```
Comment (Jedddy) [inline bot/cogs/currency_converter.py:85]: Eto pede amount = float(amount) nalang with or without a ".", kasi iroround din naman to 2 decimal places

Comment (Jiseeeh) [reply bot/cogs/currency_converter.py:85]: so it's okay for **24** to be displayed as **24.0** nalang? akala ko kasi pangit tingnan HAHHAA

Comment (kylecdrck) [reply bot/cogs/currency_converter.py:85]: Yea it's better for consistency

**Review by kylecdrck — APPROVED** [review]
lgtm

**Review by Jedddy — APPROVED** [review]

**Pull request merged**